### PR TITLE
US107 edit ce details

### PR DESCRIPTION
**Pull request by emilioward**
Run against:
 https://github.com/ONSdigital/response-operations-ui/pull/111
 https://github.com/ONSdigital/ras-backstage/pull/88

### CODE REVIEW

**Comment by emilioward** [comment]
[US107: INT Edit Collection Exercise details](https://trello.com/c/a3s4Yy3x/61-us107-int-edit-collection-exercise-details)

File: acceptance_tests/features/steps/edit_collection_exercise_details.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+@when('they edit/amend the details')
+def edit_collection_exercise_details(_):
+    edit_collection_exercise_details_form.edit_period('201906')
+    edit_collection_exercise_details_form.edit_user_description('12 June 2019')
+    edit_collection_exercise_details_form.click_save()
+
+
+@then('they can view the updated details')
+def view_updated_collection_exercise_details(context):
+    collection_exercises = collection_exercise.get_collection_exercises()
```
Comment (jcox-dev) [inline acceptance_tests/features/steps/edit_collection_exercise_details.py]: Does there need to be a wait here or are the changes from CE instantaneous?

Regardless, the tests pass locally when running against the branched services 👍

Comment (emilioward) [reply acceptance_tests/features/steps/edit_collection_exercise_details.py]: It seemed to be instantaneous when I was testing it so I don't believe it needs a wait

File: acceptance_tests/features/steps/edit_collection_exercise_details.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    for row in context.table:
+        collection_exercises_by_period = next(filter(lambda ce: ce['exercise_ref'] == row['period'],
+                                                     collection_exercises))
+        assert collection_exercises_by_period['exercise_ref'] == "201906"
+        assert collection_exercises_by_period['user_description'] == "12 June 2019"
+
+
+@then('they cannot edit the collection exercise period')
+def edit_period_icon_disappears_if_period_cannot_be_edited(_):
+    browser.find_by_id("edit-collection-exercise-period")
```
Comment (andrewmil) [inline acceptance_tests/features/steps/edit_collection_exercise_details.py]: The browser commands should be in the `pages` file and called as a function from here

Comment (andrewmil) [inline acceptance_tests/features/steps/edit_collection_exercise_details.py]: This step isn't required

**Pull request merged**